rocketmq-console/src/main/java/org/apache/rocketmq/console/service/impl/DashboardCollectServiceImpl.java

```diff
@@ -45,7 +45,7 @@
 public class DashboardCollectServiceImpl implements DashboardCollectService {
 
     @Resource
-    private RMQConfigure rmqConfigure;
+    private RMQConfigure rMQConfigure;
```

Comment:
It's a little wordy for rmq or rMQ..., you can leave it out. Rename it configure directly

Reply:
> It's a little wordy for rmq or rMQ..., you can leave it out. Rename it configure directly

We defined rMQConfigure in BaseControllerTest, and all subsequent unit test classes inherited this BaseControllerTest. In order to reuse this, we changed it to rMQConfigure. 
In order to maintain unity，I'll revise it，all references in projects are directly modified to configure.